src/Interval.cpp

```diff
@@ -99,17 +99,13 @@ std::string Interval::serialize () const
 std::string Interval::json () const
 {
   std::stringstream out;
-  out << '{';
+  out << "{\"id\":" << id;
 
   if (is_started ())
-    out << "\"start\":\"" << start.toISO () << "\"";
+    out << ",\"start\":\"" << start.toISO () << "\"";
```

Comment:
Just a note that going forward @lauft has asked for there to be curly braces around all code blocks in new and modified code.

See:
https://github.com/GothenburgBitFactory/timewarrior/pull/269#discussion_r367937920

Reply:
Wasn't aware of that, will change.

I saw the one statement without braces and thought that was the way this project did them.